a11y.js

```diff
@@ -9,30 +9,51 @@ const isSmoke = true;
 const baseUrl = 'http://localhost:7080';
 
 /**
- * Suppress iframe errors -> '//div[@id='root']/main/div/div/div/div/iframe'
- * To be fixed in https://github.com/bbc/bbc-a11y/issues/298
  *
  * Suppress errors in scrollable navigation for RTL languages -> '//div[@id='root']/header/nav/div/div[1]/div/ul'
  * To be fixed in https://github.com/bbc/simorgh/issues/5222
  */
 
+const mediaEmbedTestsToSuppress = [
+  // * Design: Content resizing: Text must be styled with units that are resizable in all browsers
+  "Text styled with px unit: //button[@id='p_audioui_backInterval']/div/div[1]",
+  "Text styled with px unit: //button[@id='p_audioui_backInterval']/div/div[2]",
+  "Text styled with px unit: //button[@id='p_audioui_forwardInterval']/div/div[1]",
+  "Text styled with px unit: //button[@id='p_audioui_forwardInterval']/div/div[2]",
+  "Text styled with px unit: //div[@id='p_audioui_backToStartButton_text']",
+  "Text styled with px unit: //div[@id='p_audioui_toLiveButton_text']",
+
+  // * Design: Content resizing: Text cannot be too small
+  "Text size too small (10px): //div[@id='p_audioui_backToStartButton_text']",
+  "Text size too small (10px): //div[@id='p_audioui_toLiveButton_text']",
+
+  // * Editorial: Indicating language: Html must have lang attribute
+  'html tag has no lang attribute: /html',
+
+  // * Forms: Labelling form controls: Fields must have labels or titles
+  "Button has no text: //button[@id='p_audioui_playpause']",
+  "Button has no text: //button[@id='p_audioui_previousButton']",
+  "Button has no text: //button[@id='p_audioui_nextButton']",
+  "Button has no text: //div[@id='p_audioui_container']/div[2]/button",
+  "Button has no text: //button[@id='p_audioui_playbackSettingsButton']",
+  "Button has no text: //div[@id='p_audioui_container']/div[4]/button",
```

Comment:
@greenc05 all of these are failing when I removed the full iframe error suppression. Do you think we should have the error suppressors more specific so we can catch stuff we can fix like the html language one and potentially the button ones? We could also see if moving to Toucan means we can remove some of these error suppressions.

Reply:
I'm not sure, would that make it more fragile if SMP update their code?

Reply:
Maybe we should just put them back and wait for Toucan?

Reply:
If they change their code and cause more issues then yeh this would fail

Reply:
Cool, lets revert for pages where we got these error then, can re-look when we get Toucan on pages with SMP